app/models/team.rb

```diff
@@ -635,6 +635,11 @@ def get_dashboard_exported_data(filters, dashboard_type)
     data
   end
 
+  # Platforms for which statistics are available (e.g., at least one media request)
+  def statistics_platforms
+    TiplineRequest.joins(:project_media).where('project_medias.team_id' => self.id).group('platform').count.keys
```

Comment:
just out of curiosity: Any reason not to use `.pluck(:platform).uniq` instead of `.group('platform').count.keys` ?

Reply:
Using `.pluck(:platform).uniq` would load all platforms into Ruby memory and calls `.uniq`, which removes duplicates at the Ruby level... it wouldn't have good performance and could even trigger a OOM killer event.

But now I'm realizing that the best alternative (performance-wise) is to use `.distinct.pluck(:platform)`, which filters unique values at the database value... let me make that change.

Reply:
> it wouldn't have good performance and could even trigger a OOM killer event.
aaaaaa! That makes sense! Thank you 🙏

Reply:
Thank you! The improvement is now in this other PR: https://github.com/meedan/check-api/pull/2232